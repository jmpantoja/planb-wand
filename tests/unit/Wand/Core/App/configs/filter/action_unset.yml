---
tasks:
  taskA:
   actionA: true
  taskB:
   actionC: true

