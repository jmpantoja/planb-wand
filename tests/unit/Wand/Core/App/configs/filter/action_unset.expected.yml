tasks:
  taskA:
    classname: \PlanB\Wand\Core\Task\Task
    description: wubba lubba dub dub
    actions:
      actionA:
        group: metainfo
        classname: \PlanB\Wand\Core\File\File
        params:
          template: '@wand.metainfo.readme.twig'
          target: README.md
          action: override # delete, create
          chmod: 0644

  taskB:
    classname: \PlanB\Wand\Core\Task\Task
    description: wubba lubba dub dub
    actions:
      actionC:
        group: metainfo
        classname: \PlanB\Wand\Core\File\File
        params:
          template: '@wand.metainfo.readme.twig'
          target: README.md
          action: override # delete, create
          chmod: 0644